priv/repo/migrations/20220425135720_add_request_time_in_chicago_tz_index_to_requests.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Tilex.Repo.Migrations.AddRequestTimeInChicagoTzIndexToRequests do
+  use Ecto.Migration
+
+  def up do
+    execute """
+      create index index_requests_on_request_time_in_chicago on requests (timezone('america/chicago', request_time))
```

Comment:
I'd suggest us to get this `america/chicago` value from the config, so other TIL instances that have configured diff TZ for them would benefit from this index as well.

```elixir
(timezone('#{Application.get_env(:tilex, :date_display_tz, "america/chicago")}', request_time))
```

Reply:
Done! I've added a commit to cover this